package-lock.json

```diff
@@ -857,6 +857,11 @@
         "through": ">=2.2.7 <3"
       }
     },
+    "a-sync-waterfall": {
```

Comment:
I wouldn't have expected any changes in this file - can you restore the original file and then run `make uninstall` and then `make install`. If it does make any changes, that's fine but please confirm this